src/_33_4_Providing_the_function_to_solve.jl

```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/env julia
```

Comment:
I think as long as there's only trackable incremental updates to the API, you shouldn't need to update the auto-generated files/regenerate them at all. Even if there's no update, regenerate them will just overwrite any fix that has to be done by hand.